pkg/api/v1/status/golbal_deployment.go

```diff
@@ -0,0 +1,50 @@
+package status
```

Comment:
[pkg/api/v1/status/golbal_deployment.go](https://github.com/mongodb/mongodb-atlas-kubernetes/pull/813/files#diff-646e8cf4e195791ee76b7092c53c6b8dd9e39a86d0f7e56c8c2f33f0fc6d154a) typo in the name of the file

Reply:
Done